webhooks.md

```diff
@@ -0,0 +1,113 @@
+# Webhooks
+
+Currently supported webhooks for operation on integrations. Action is triggered everytime when hotel updates the integration.
+
+### Webhook actions
+
+* [Integration created](webhooks.md#integration-created-data) - Whenever integration is created.
+* [Integration disabled](webhooks.md#integration-disabled-data) - Whenever integration is disabled.
+* [Integration enabled](webhooks.md#integration-enabled-data) - Whenever integration is enabled.
+* [Integration deleted](webhooks.md#integration-deleted-data) - Whenever integration is deleted.
+* [Integration canceled](webhooks.md#integration-canceled-data) - Whenever integration is canceled.
+* [Integration reinstated](webhooks.md#integration-reinstated-data) - Whenever integration is reinstated.
+
+## Message
+
+```javascript
+{
+  "Action": "IntegrationCreated",
+  "Data": {
+    "Enterprise": {
+      "Id": "8865aa96-f62d-4f9b-a912-ab2100f60f42",
+      "Name": "Sample Chain Hotel 1"
+    },
+    "Requestor": null,
+    "AccessToken":"9E5E84E9974D4F169662AB2200F27CB1-00B343A0DDA725CACAC028E38E3EABF",
+    "CreatedUtc": "2019-12-13T14:42:52Z",
+    "IsEnabled": true,
+    "Integration": {
+      "Id": "9e5e84e9-974d-4f16-9662-ab2200f27cb1",
+      "Name": "WebhookTEST"
+    }
+  }
+}
+
+```
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Action` | string [Webhook action](webhooks.md#Webhook-actions) | required | Type of action |
+| `Data` | object | required | Webhook [Data](webhooks.md#Data). |
+
+## Data
+
+Structure of data varies depending on the [Webhook](webhooks.md#Webhooks) action:
+
+* `IntegrationCreated` - [Integration created data](webhooks.md#integration-created-data)
+* `IntegrationDisabled` - [Integration disabled data](webhooks.md#integration-disabled-data)
+* `IntegrationEnabled` - [Integration enabled data](webhooks.md#integration-enabled-data)
+* `IntegrationDeleted` - [Integration deleted data](webhooks.md#integration-deleted-data)
+* `IntegrationCanceled` - [Integration canceled data](webhooks.md#integration-canceled-data)
+* `IntegrationReinstated` - [Integration reinstated data](webhooks.md#integration-reinstated-data)
+
+### Integration created data
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Enterprise` |[Enterprise](webhooks.md#Enterprise)| required | Commercial chain of the property |
+| `Requestor` | [Requestor](webhooks.md#Requestor) | required |  Person requesting action. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `CreatedUtc` | string | required | Creation date and time of the integration. |
+| `IsEnabled` | bool | required | Integration enabled by default. |
```

Comment:
Whether integration is enabled. "by default" does not make much sense IMHO, this is no longer default, this is right after creation.